content/en/ecosystem/adopters.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Adopters
+description: Organizations that have adopted OpenTelemetry
+# prettier-ignore
+spelling: cSpell:ignore Dapr Datenrettungsdienste Globale Logicmonitor Logz Wandera Zocdoc
+# All spelling entries must be on a single line
+---
+
+A non-exhaustive, alphabetized list of organizations that have adopted
+OpenTelemetry.
```

Comment:
I suggest adding:
  * a comment that people can add their org to the list
  * a link on how / where to do that
  * mention a requirement that the submission must include a link to a blog post that explains how the org is using OTEL

Reply:
Done